packages/pingcap-ui/src/index.js

```diff
@@ -11,11 +11,23 @@ import * as footerData from './components/footer/footer.data';
 import * as headerData from './components/header/header.data';
 // Utils
 import * as headerUtils from './components/header/header.utils';
-
-export const data = {
-  footer: footerData,
-  header: headerData,
-};
+import { nav } from './utils';
+
+export function getData(urlPrefixRegexp) {
```

Comment:
Not a big deal, but personally I would appreciate it if we could prioritise ES6 function.

```javascript
export const getData = urlPrefix => {
  // ...
}
```

Please also think about a more suitable argument name, as I suggested in [Core.component.js](https://github.com/pingcap/tug-website/pull/36/files#diff-1ae3e11c5f4017444a47a6733e1b04bd2088c3ee9d328944446c197bf51cb53d).

Maybe at the initial stage, we could only consider bashPath, unless we will have other use cases based which are on regular expressions? In that case, the argument would be renamed to `bashPath`?